docker/README.md

```diff
@@ -94,20 +78,9 @@ and use variables on `compose.yml`:
       - config/server.properties
       # ...
       - --override
-      - rsm.config.s3.client.aws_access_key_id=${AWS_ACCESS_KEY_ID}
+      - rsm.config.s3.public_key_pem=/kafka/plugins/public.pem
       - --override
```

Comment:
1. The names are incorrect (should be `rsm.config.encryption.public.key.file`, etc.)
2. `rsm.config.encryption=true` is needed.
3. Better to override using env variables